### ISSUE TITLE
video resize

### ISSUE THREAD

**Issue by mrbdrm** [skiptirengu/media-split#13]
im using this great package and was wandering: is there an ability to resize the splitted videos?

**Comment by sugiruu** [comment]
Currently there is no way to do that.
I'll add a new option to allow passing additional parameters to the ffmpeg command for each section.

**Comment by sugiruu** [comment]
Fixed with version 5.4.0

- `-p:i` will add arguments to FFMpeg's input file options
- `-p:o` will add arguments to FFMpeg's output file options

Now you can do something like the following:
```bash
media-split -i myvideo.avi -f mp4 -t sections.txt -o /home/user/Videos -p:o "'-vf scale=320:-1'"
```

If you're having issues with ffmpeg, try double quoting the additional parameters.
